src/v2/Apps/Order/Components/SavedAddresses.tsx

```diff
@@ -89,15 +99,93 @@ const defaultAddressIndex = addressList => {
   return `${indexOfDefaultAddress > -1 ? indexOfDefaultAddress : 0}`
 }
 
-interface Props {
-  me: SavedAddresses_me
-  onSelect: (string) => void
-  handleClickEdit: (number) => void
-}
+const SavedAddresses: React.FC<SavedAddressesProps> = props => {
+  const [showAddressModal, setShowAddressModal] = useState(false)
+  const { onSelect, handleClickEdit, me, inCollectorProfile, relay } = props
+  const addressList = me?.addressConnection?.edges ?? []
+  const handleModifyAddressModal = () => {
+    setShowAddressModal(!showAddressModal)
+  }
+  const collectorProfileAddressItems = addressList.map((address, index) => {
+    const isDefaultAddress = address?.node?.isDefault
```

Comment:
If `address` is an empty array then there'd be no looping; if `address` isn't empty, then surely its a connection with a `node`, so we probably don't need the `?` on `address` (since its defined) or `node`, because we're requesting a connection and node always comes with it (unless there's a bug in metaphysics).